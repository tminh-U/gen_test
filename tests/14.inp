8
15 10 11 20 12 12 13 14 
10
4 8
4 7
2 4
1 5
4 4
3 7
4 4
7 7
4 8
2 3
