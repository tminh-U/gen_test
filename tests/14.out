17
9
8
14
2
18
2
2
17
4
